packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -180,6 +180,15 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
     );
   }
 
+  protected okToAddMore(): boolean {
+    const { items } = this.state;
+    const { itemLimit } = this.props;
+    if (typeof itemLimit !== 'undefined' && items.length === itemLimit) {
```

Comment:
just a thought.. performance wise... 
checking for undefined on itemLimit would exit early and skip checking the length of items. a micro optimization granted, but is it worth keeping in?